docker/caffe2/jenkins/common/install_rocm.sh

```diff
@@ -73,6 +73,9 @@ else
   exit 1
 fi
 
+# Enable the usage of LD.Gold for linking.
+ln -s `which gold` /usr/bin/ld
```

Comment:
The more "official" way to do this is to use `update-alternatives` to set ld to a different implementation. There are a few examples of this in the jenkins docker configs already.

Reply:
This approach will also fail since `/usr/bin/ld` exists.